padd.sh

```diff
@@ -1600,26 +1613,12 @@ Update() {
 
         padd_script_path=$(realpath "$0")
 
-        if which wget > /dev/null 2>&1; then
-            echo "${check_box_info} Downloading PADD update via wget ..."
-            if wget -qO "${padd_script_path}" https://install.padd.sh > /dev/null 2>&1; then
-                echo "${check_box_good} ... done. Restart PADD for the update to take effect"
-            else
-                echo "${check_box_bad} Cannot download PADD update via wget"
-                echo "${check_box_info} Go to https://install.padd.sh to download the update manually"
-                exit 1
-            fi
-        elif which curl > /dev/null 2>&1; then
-            echo "${check_box_info} Downloading PADD update via curl ..."
-            if  curl -sSL https://install.padd.sh -o "${padd_script_path}" > /dev/null 2>&1; then
-                echo "${check_box_good} ... done. Restart PADD for the update to take effect"
-            else
-                echo "${check_box_bad} Cannot download PADD update via curl"
-                echo "${check_box_info} Go to https://install.padd.sh to download the update manually"
-                exit 1
-            fi
+        echo "${check_box_info} Downloading PADD update ..."
+
+        if  curl -sSL https://install.padd.sh -o "${padd_script_path}" > /dev/null 2>&1; then
+            echo "${check_box_good} ... done. Restart PADD for the update to take effect"
```

Comment:
PADD could restart itself. Sure, this would recurse deeper and deeper but it will take hundreds of updates until we hit the recursion depth limit (defaults to 256 IIRC).

Reply:
Do you suggest it should restart itself?  I'm not sure this is a good idea.

Reply:
The `update()` is only called when invoked from CLI with `padd --update`. The `main()` is not running at all. So PADD might not even be 'really' running. We likely would need to pass arguments (such as `--server`) to the update function in order to properly start PADD as the user want's it to start. We would also need to offer the choice not to (re)start PADD.
Or PADD is running in a different shell - which would need to be killed first. 

I think there is little gain and a lot of pitfalls for such a small change. We already notify users if there is a update available and we offer a way to run the update with a single command. (Re)starting PADD manually seems fine to me.